src/modules/footer.js

```diff
@@ -0,0 +1,11 @@
+const font = () => {
+  const footer = document.getElementById('footer');
+
+  footer.innerHTML += `<hr class="foot-line">
+<p class="foot-item">Created by microverse under CC license</p>
+`;
+};
+
+/* eslint-disable */
```

Comment:
- [ ] Same issue as discussed above, please enable linters and try solving the issues involved.